backend/src/main/java/com/huellapositiva/domain/service/EmailCommunicationService.java

```diff
@@ -50,8 +52,8 @@ public void sendRegistrationConfirmationEmail(EmailConfirmation emailConfirmatio
      */
     public void sendRevisionRequestEmail(ProposalRevisionRequestEmail proposalRevisionRequestEmail) {
         EmailTemplate emailTemplate = templateService.getProposalRevisionRequestTemplate(proposalRevisionRequestEmail);
-        Email email = Email.createFrom(proposalRevisionRequestEmail, emailTemplate, from);
-        emailService.sendEmail(email);
+        EmailMessage emailMessage = EmailMessage.createFrom(from, proposalRevisionRequestEmail.getEmailAddress(), "Revisión de nuevas convocatorias requerida", emailTemplate);
+        emailService.sendEmail(emailMessage);
```

Comment:
Would it be better to create a value object or create a constant? It's too long the 55 line.